apps/docs/src/app/core/component-docs/carousel/examples/carousel-loading-content.component.ts

```diff
@@ -0,0 +1,11 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-carousel-loading-content',
+    templateUrl: './carousel-loading-content.component.html',
+    styles: ['fd-carousel:focus {outline: 1px dotted}', 'img {pointer-events: none;}'],
```

Comment:
1. This styling is for working swipe behaviour in non-touch devices. Putting this style inside component will restrict user to have clicks and other pointer events.(because of using pointer-events: none). I have kept a note on documentation.

for touch devices this is not an issue.

2. for focus border styling, fiori does not define focus for carousel. so by default focus is different on different browsers. developers using this component can define whatever focus they need. Putting this inside component will restrict developers.

Hope this is fine.

Reply:
1. So if this is a "custom" behavior it should not be implemented into all of the examples. It should be only in the example when you mention about how to achieve this behavior.

2. If this is not specified, do not patch it on documentation level. You are creating a false visuals of the component and users will be confused, why after implementing carousel on their own, the focus is different. Could you create a ticket for carousel focus in Fundamental Styles?

Reply:
created issue in fundamental-styles. https://github.com/SAP/fundamental-styles/issues/1901
removed style patching from examples, kept in 1 horizontal and vertical carousels.